c/array-patterns/array10_pattern.c

```diff
@@ -29,19 +29,19 @@ void __VERIFIER_assert(int cond) { if(!(cond)) { ERROR: __VERIFIER_error(); } }
 extern int __VERIFIER_nondet_int() ;
 extern short __VERIFIER_nondet_short() ;
 
-short ARR_SIZE ;
+signed long long ARR_SIZE ;
 
 int main()
 {
-	ARR_SIZE = __VERIFIER_nondet_short() ;
+	ARR_SIZE = (signed long long)__VERIFIER_nondet_short() ;
 	__VERIFIER_assume(ARR_SIZE > 0) ;
 
 	int array1[ARR_SIZE] ;
 	int array2[ARR_SIZE] ;
 	int count = 0, num = -1 ;
         signed long long sum = 0 ;
 	int temp ;
-	short index ;
+	signed long long index ;
```

Comment:
I'd rather make `num` a `signed long long` - that would  avoid the typecast in line 55 (but have the same effect on the possibly-overflowing computations in lines 58 and 59.

Reply:
Do you want me to go ahead and do these changes? I can do that. However, the intention of these benchmarks are not for checking overflows. Possibly, I can have a more detailed look into these programs later from overflow perspective and submit benchmarks having overflow property too!!